frontend/packages/console-app/src/components/nav/ClusterMenuToggle.tsx

```diff
@@ -52,7 +52,7 @@ const ClusterMenuToggle = (props: {
       onClick={() => onToggle(!isOpen)}
       isExpanded={isOpen}
       variant="plainText"
```

Comment:
```suggestion
```
If we're trying to be consistent with console.redhat.com, then `Menu` toggle would have a standard display with `hover` and `active` blue border bottom. This can be achieved by removing this and modifying some styles in the _masthead.scss

Reply:
Yeah.  I chose not to include that since we have a regular dropdown and not a full-page one.  Seemed like it might be weird?

Reply:
@sg00dwin and I discussed this, and are opting to leave it as is as removing requires more CSS overrides.